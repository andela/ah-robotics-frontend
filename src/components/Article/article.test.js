import React from 'react';
import ArticleComponent from './index';

describe('single article view tests', () => {
it('it should render single article view', () => {
  const wrapper = shallow(<ArticleComponent article={{}} isFetching={false} />);
  expect(wrapper.length).toBe(1);
  expect(wrapper.find('container')).toBeDefined();
});
it('it should match snapshot', () => {
  const wrapper = shallow(<ArticleComponent article={{}} isFetching={false} />);
  expect(wrapper).toMatchSnapshot();
});
});
