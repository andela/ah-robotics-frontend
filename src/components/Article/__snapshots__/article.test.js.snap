// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`single article view tests it should match snapshot 1`] = `
<Segment
  basic={true}
  loader={false}
>
  <Container
    style={
      Object {
        "marginTop": "7em",
      }
    }
    text={true}
  >
    <Header
      as="h1"
    />
    <Image
      as="img"
      className="article-image"
      src="https://source.unsplash.com/random/720x580"
      ui={true}
    />
    <p />
    <span
      className="article-span"
    />
    | 
    <span
      className="article-span"
    >
      Sunday 24th February 2019
    </span>
  </Container>
</Segment>
`;
